src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+
+    {person.isMarried &&
+      (person.sex === 'm' ? (
+        <p className="Person__partner">{person.partnerName} is my wife</p>
+      ) : (
+        <p className="Person__partner">{person.partnerName} is my husband</p>
+      ))}
+
+    {!person.isMarried && <p className="Person__partner">I am not married</p>}
```

Comment:
- Use the ternary operator when you have two options(person.isMarried) like in this case.
- <p className="Person__partner"> is the same in all cases, don't repeat it, only content is different 
- `{person.partnerName} is my ` here the content is almost the same, the only different is in word husband/wife, so you can set only it conditonally